app/views/user_mentee_applications/new.html.erb

```diff
@@ -1,66 +1,83 @@
-<dl class="text-center mb-2">
-  <dt><%= current_user.full_name %></dt>
-</dl>
+<h1 class="text-center font-bold text-3xl mb-5">
+  Welcome <%= current_user.first_name %>!
+</h1>
 
-<%= form_with model: @user_mentee_application, local: true, html: { class: 'flex flex-col' } do |form| %>
+<%= form_with model: @user_mentee_application, local: true, html: { class: 'max-w-prose mx-auto' } do |form| %>
+  <div class='flex flex-col ml-5 mr-5'>
+    <%= render UserMenteeApplication::SectionComponent.new(header_text: 'Tell us about your journey') do %>
+      <div class="flex flex-col mb-5">
+        <%= form.label :learned_to_code, 'How did you learn to code?', class: 'font-bold' %>
+        <%= form.text_area :learned_to_code, class: 'form-input border-2 mb-2', required: true %>
+      </div>
+      <div class="flex flex-col">
+        <%= form.label :project_experience, "Describe a project you’ve worked on that you especially loved or are proud of.",  class: 'font-bold' %>
+        <%= form.text_area :project_experience, class: 'form-input border-2 mb-2', required: true %>
+      </div>
+    <% end %>
 
-  <div class="flex flex-col">
-    <%= form.label :city, class: 'font-bold' %>
-    <%= form.text_field :city, class: 'form-input' %>
-  </div>
+    <%= render UserMenteeApplication::SectionComponent.new(header_text: 'Tell us about your interest in the Agency of Learning') do %>
+      <div class="flex flex-col mb-5">
+        <%= form.label :reason_for_applying, 'Why are you here?', class: 'font-bold' %>
+        <%= form.select :reason_for_applying, options_for_select(UserMenteeApplicationsHelper::REASONS_FOR_APPLYING), class: 'form-input mb-2', required: true %>
```

Comment:
The styles on this `form.select` actually aren't being applied. You can check by inspecting the `<select>` tag's markup and seeing that no classes are there.

This is because of the kinda dumb API they have for this method lol: https://api.rubyonrails.org/v7.0.7/classes/ActionView/Helpers/FormBuilder.html#method-i-select

You basically have to write it like:
```erb
<%# Note the empty options hash among the args %>
form.select :reason_for_applying, options_for_select(reasons), {}, class: 'these classes work', required: true
```